peregrine/include/glo_ca_code.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2016 Swift Navigation Inc.
+#
+# Contact: Pasi Miettinen <pasi.miettinen@exafore.com>
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+import types
+import sys
+import numpy as np
+
+
+def generate_glo_ca_code():
+  """
+  Function generates PRN sequence for Glonass.
+  All GLONASS satellites use the same C/A-code,
+  generated by a 9-bit shift register.
+  """
+  code = '111111111'
+  output = np.zeros(511, np.int8)
+  for i in xrange(len(output)):
+    if '0' == code[6]:
+      output[i] = 1
+    else:
+      output[i] = -1
+    if int(code[4]) ^ int(code[8]):
+      code = '1' + code[:8]
+    else:
+      code = '0' + code[:8]
+  return output
+
+
+def readonly(value):
```

Comment:
Could we make this more similar to the other code generation modules? See the following:
- https://github.com/adel-mamin/peregrine/blob/peregrine-l2c-update/peregrine/include/generateL2CMcode.py
- https://github.com/swift-nav/peregrine/blob/master/peregrine/include/generateCAcode.py

Steps should be:
- Remove `readonly`, `glo_ca_code`, and `tmp`
- Add `GLOCode = generate_glo_ca_code` to the bottom of the module
- Rename file to `generateGLOcode.py`